app/assets/stylesheets/watch-listen.css.scss

```diff
@@ -0,0 +1,27 @@
+.watch-listen {
+  .video-wrapper {
+    height: 0;
+    margin: 0 0 ($gutter / 2) 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 $gutter / 2` instead of `0 0 $gutter / 2 0`